### PR TITLE
Update route table names based on shared_public_route_table value.

### PR DESCRIPTION
**Pull request by dogfish182**
If we use multiple route tables they shouldn't share the same name.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -125,7 +125,9 @@ resource "aws_route_table" "public" {
   vpc_id = aws_vpc.default.id
   tags = merge(
     var.tags,
-    { "Name" = "${var.prepend_resource_type ? "route-table-" : ""}${var.name}-public" }
+    {
+      "Name" = var.shared_public_route_table ? "${var.prepend_resource_type ? "route-table-" : ""}${var.name}-public" : "${var.prepend_resource_type ? "route-table-" : ""}${var.name}-public-${local.az_ids[count.index]}"
+    }
```
Comment (svanharmelen) [inline main.tf:130]:
```suggestion
    {
        "Name" = "${var.prepend_resource_type ? "route-table-" : ""}${var.name}-public${var.shared_public_route_table ?  : "-${local.az_ids[count.index]}"}"
    }
```

**Review by svanharmelen — CHANGES_REQUESTED** [review]
Maybe make it a little less repeating?

File: main.tf (hunk 1 above)
Comment (svanharmelen) [reply main.tf:130]: I didn't test/verify this, but I guess you get the point?

Comment (dogfish182) [reply main.tf:130]: will verify now looks good

**Review by svanharmelen — APPROVED** [review]
👍🏻

**Pull request merged**